src/app/EventManagement.cpp

```diff
@@ -858,6 +858,15 @@ bool CircularEventBuffer::IsFinalDestinationForPriority(PriorityLevel aPriority)
     return !((mpNext != nullptr) && (mpNext->mPriority <= aPriority));
 }
 
+/**
+ * @brief
+ *  Implements CircularEventBuffer::OnInit(TLVWriter) for circular buffers.
```

Comment:
That part is clear.  We should document _why_.  As in, document that TLVCircularBuffer::OnInit can modify the state of the buffer, and we don't want that behavior here.  We want to make sure we don't change our state.  So maybe something like:
```suggestion
 * TLVCircularBuffer::OnInit can modify the state of the buffer, but we don't want that behavior here.
 * We want to make sure we don't change our state, and just report the currently-available space.
```

Reply:
updated, thanks